.github/workflows/waldo_sessions.yml

```diff
@@ -0,0 +1,57 @@
+name: Upload builds to Waldo
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+    
+jobs:
+  build:
+    runs-on: macos-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: "Checkout code"
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Setup Java JDK
+        uses: actions/setup-java@v3.11.0
+        with:
+          distribution: 'zulu'
+          java-version: 11
+
+      - name: Setup Flutter
+        uses: subosito/flutter-action@v2
+        with:
+          #channel: stable
+          cache: true
+          flutter-version: ${{ steps.flutter-version.outputs.FLUTTER_VERSION }}
```

Comment:
You forgot to include the step which outputs the flutter version we want

Reply:
Ooops you're right, that's a bad copy/paste.
That should now be OK (line 28)